librabbitmq/amqp_private.h

```diff
@@ -48,7 +48,17 @@
 #include "amqp_framing.h"
 #include <string.h>
 
-#ifdef _WIN32
+#if defined (__MINGW32__) || defined (__MINGW64__)
```

Comment:
Instead of duplicating this whole block, modify line 51 so that it also covers the mingw* cases, then correct the capitalization of `#include <winsock2.h>`

The [win32 docs](https://msdn.microsoft.com/en-us/library/windows/desktop/ms737629(v=vs.85).aspx) seem to indicate that all-lowercase is the correct capitalization.